src/redux/categories/categories.js

```diff
@@ -0,0 +1,19 @@
+// Actions
+const CHECK_STATUS = 'redux/categories/';
```

Comment:
- [ ] Kindly fix the action type to follow the duck pattern like this: `const CHECK_STATUS =' bookstore/categories/CHECK_STATUS'`